Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -13,28 +13,35 @@ var restObj = new restClient.RestCallbackClient(httpObj);
 
 var defaultAuthUrl = 'https://login.windows.net/';
 var azureApiVersion = 'api-version=2016-08-01';
+var azureContainerRegistryApiVersion = "api-version=2017-03-01";
```

Comment:
@Ajay-MS is there a support for Azure stack?

Reply:
Endpoint object initialization takes care of National Cloud/Azure Stack  URLs. Hence changes are compatible for the azure stack and national cloud. We might have to change API version for azure stack later as currently, ACR support is not there with Azure Stack and not sure about API version.   @pavanadepu2  Can you please verify changes for US government cloud once.